gio/src/async_initable.rs

```diff
@@ -0,0 +1,51 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::traits::AsyncInitableExt;
+use crate::AsyncInitable;
+use crate::Cancellable;
+use glib::object::IsA;
+use glib::object::IsClass;
+use glib::value::ToValue;
+use glib::{Cast, Object, StaticType, Type};
+
+impl AsyncInitable {
+    #[doc(alias = "g_async_initable_new_async")]
+    #[allow(clippy::new_ret_no_self)]
+    pub fn new_async<
```

Comment:
Should probably have a _future variant as well?

Reply:
It should... but that always makes my head spin :)